WooCommerce/Classes/POS/Models/PointOfSaleAggregateModel.swift

```diff
@@ -92,6 +92,11 @@ protocol PointOfSaleAggregateModelProtocol {
 // MARK: - ItemList
 @available(iOS 17.0, *)
 extension PointOfSaleAggregateModel {
+    @MainActor
+    func toggleItemType() async {
```

Comment:
Would you say we should be explicitly of what type we pass through? I went with "toggle" for simplicity for now, but I'm always wary of ending in the wrong toggle state if we don't pass the property through here 🤔

Reply:
> Would you say we should be explicitly of what type we pass through?

Yes 👍 We already now we'll have at least a third option - customers, so the choice can be represented with enum.